.github/workflows/release.yml

```
@@ -109,8 +113,14 @@ jobs:
         name: 'Prepare CDN release'
         run: echo "versions=$(npm run --silent ci:prepare-release)" >> ${GITHUB_OUTPUT}
 
+      - name: generate build provenance
+        if: ${{ always() && hashFiles('packages/rum/dist/bundles/*.js')
```

Sorry, there were missing `}}` in the suggestion.

```suggestion
        if: ${{ always() && hashFiles('packages/rum/dist/bundles/*.js') }}
```